lib/improver/cli/wxcode.py

```diff
@@ -56,6 +61,11 @@ def process(*cubes: cli.inputcube,
     """
     from iris.cube import CubeList
     from improver.wxcode.weather_symbols import WeatherSymbols
+    from improver.wxcode.wxcode_utilities import interrogate_decision_tree
+
+    if extended_help is not None:
```

Comment:
Hmm.. nice idea. How about if we make `*cubes` have a default of `None`, and then change this `if not cubes`?

Reply:
(and then just remove the `extended_help` variable - i.e: spit out help if we get no inputs)